olive/passes/onnx/transformer_optimization.py

```diff
@@ -94,30 +94,62 @@ def sd_model_types():
         return ("unet", "vae", "clip")
 
     @staticmethod
-    def _set_sd_fusion_options(run_config: Dict[str, Any], use_float16: bool):
+    def _set_sd_fusion_options(run_config: Dict[str, Any], pass_config: Dict[str, Any]):
         """Configures fusion options for stable diffusion models"""
         import onnxruntime as ort
 
-        input_model_type = run_config["model_type"]
+        ort_version = version.parse(ort.__version__)
+        is_ort_1_13_or_older = ort_version < version.parse("1.14.0")
+        # is_ort_1_14 = ort_version >= version.parse("1.14.0") and ort_version < version.parse("1.15.0")
```

Comment:
stale deletable comment?

Reply:
I'll delete this when I remove directml_future to avoid spinning the CI again.